jwst/datamodels/tests/test_fits.py

```diff
@@ -4,7 +4,8 @@
 import numpy as np
 import pytest
 
-from jwst.datamodels import JwstDataModel, RampModel
+import jwst.datamodels
```

Comment:
As elsewhere in the package, this import is preferred:

```suggestion
from jwst import datamodels
```